.travis.yml

```diff
@@ -1,65 +1,50 @@
 language: python
 
-sudo: false
+dist: xenial
 
 matrix:
   include:
     - python: 3.5
       env: TOX_ENV='flake8'
     - python: 3.5
       env: TOX_ENV='isort'
-      # Django 1.11
-    - python: 3.4
-      env: DJANGO='dj111' CMS='cms35' FE=1
     - python: 3.5
-      env: DJANGO='dj111' CMS='cms36' FE=1
-    - python: 3.6
-      env: DJANGO='dj111' CMS='cms37' FE=1
-    # Django 2.1
-    - python: 3.6
-      env: DJANGO='dj21' CMS='cms36' FE=1
-    - python: 3.6
-      env: DJANGO='dj21' CMS='cms37' FE=1
+      env: TOX_ENV='frontend'
     # Django 2.2
+    - python: 3.5
+      env: DJANGO='dj22' CMS='cms37'
     - python: 3.6
-      env: DJANGO='dj22' CMS='cms37' FE=1
+      env: DJANGO='dj22' CMS='cms37'
     - python: 3.7
-      env: DJANGO='dj22' CMS='cms37' FE=1
-      dist: xenial
-      sudo: true
+      env: DJANGO='dj22' CMS='cms37'
     - python: 3.8
       env: DJANGO='dj22' CMS='cms37'
-    # Django 3.0
+    # Django 3.0, always run the lowest supported version
     - python: 3.6
       env: DJANGO='dj30' CMS='cms37'
-    - python: 3.7
```

Comment:
@FinalAngel What's the thinking behind not having Python 3.7+ for the cutting edge Django versions? 

It would be good to know how this has been decided so that any changes I oversee follow the same rules. 

I would think that the current stable (LTS 2.2) and the latest cutting edge 3.1 would have a fairly comprehensive test suite to ensure stability (LTS / 2.2) and to foresee any issues coming ahead (cutting edge).

Reply:
The thinking was to have a partial match and only introduce additional tests when reported. We did did with all the other addons in the past as well. We rarely had issues with other custom versions.